core-bundle/contao/library/Contao/Model.php

```diff
@@ -223,6 +223,20 @@ public function cloneOriginal()
 		return $clone;
 	}
 
+	/**
+	 * Clone a model with all data and prevent saving
+	 *
+	 * @return static The model
+	 */
+	public function cloneDetached()
+	{
+		$clone = clone $this;
+		$clone->setRow($this->row());
```

Comment:
```suggestion
		$clone->arrData[static::$strPk] = $this->arrData[static::$strPk];
```
This would improve the performance of the cloning operation and should have the exact same result.